lib/git_reflow/git_server/pull_request.rb

```diff
@@ -3,6 +3,24 @@ module GitServer
     class PullRequest
       attr_accessor :description, :html_url, :feature_branch_name, :base_branch_name, :build_status, :source_object, :number
 
+      APPROVAL_REGEX = /(?i-mx:lgtm|looks good to me|[:\\+1:]|:thumbsup:|:shipit:)/
```

Comment:
there also appears to be an issue matching the `:+1:` with this regex.  Please check this on [Rubular](http://www.rubular.com/).  Updating it to this seemed to match all as expected.

`(?i-mx:lgtm|looks good to me|:\+1:|:thumbsup:|:shipit:)`